src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Clear cache pools.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolClearCommand extends ContainerAwareCommand
```

Comment:
> That's written, and reviewers have this very important policy in mind all the time.

Come on, I saw many times BC break (technically) in Symfony in classes like commands, listeners, etc. But it was done with comment like "well, it's very unlikely that someone would extend this, so why not".

Further, final allows you to change the command later without even technical BC break.

You can agree with this BC, but it doesn't look like a right way to deal with the command. You'd waste your time on bad designed decisions of other developers.